src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -23,53 +22,53 @@
 
 (defn view
   []
-  (let [selected-tab (reagent/atom first-tab-id)]
-    (fn []
-      (let [{:keys [name color formatted-balance
-                    watch-only?]} (rf/sub [:wallet/current-viewing-account])
-            customization-color   (rf/sub [:profile/customization-color])]
-        [rn/view {:style {:flex 1}}
-         [account-switcher/view
-          {:type     :wallet-networks
-           :on-press #(rf/dispatch [:wallet/close-account-page])}]
-         [quo/account-overview
-          {:container-style     style/account-overview
-           :current-value       formatted-balance
-           :account-name        name
-           :account             (if watch-only? :watched-address :default)
-           :customization-color color}]
-         (when (ff/enabled? ::ff/wallet.graph) [quo/wallet-graph {:time-frame :empty}])
-         (when (not watch-only?)
-           [quo/wallet-ctas
-            {:container-style style/cta-buttons
-             :send-action     (fn []
-                                (rf/dispatch [:wallet/clean-send-data])
-                                (rf/dispatch [:wallet/wizard-navigate-forward
-                                              {:start-flow? true
-                                               :flow-id     :wallet-send-flow}]))
-             :receive-action  #(rf/dispatch [:open-modal :screen/wallet.share-address
-                                             {:status :receive}])
-             :buy-action      #(rf/dispatch [:show-bottom-sheet
-                                             {:content buy-token/view}])
-             :bridge-action   (fn []
-                                (rf/dispatch [:wallet/clean-send-data])
-                                (rf/dispatch [:wallet/wizard-navigate-forward
-                                              {:start-flow? true
-                                               :flow-id     :wallet-bridge-flow}]))
-             :swap-action     (when (ff/enabled? ::ff/wallet.swap)
-                                #(rf/dispatch [:wallet/start-swap]))}])
-         [quo/tabs
-          {:style            style/tabs
-           :size             32
-           :default-active   @selected-tab
-           :data             (tabs-data watch-only?)
-           :on-change        (rn/use-callback (fn [tab] (reset! selected-tab tab)))
-           :scrollable?      true
-           :scroll-on-press? true}]
-         [tabs/view {:selected-tab @selected-tab}]
-         [quo/floating-shell-button
-          {:jump-to
-           {:on-press            #(rf/dispatch [:shell/navigate-to-jump-to])
-            :customization-color customization-color
-            :label               (i18n/label :t/jump-to)}}
-          style/shell-button]]))))
+  (let [selected-tab          (or (rf/sub [:wallet/account-tab]) first-tab-id)
```

Comment:
selected tab is stored in the reframe db so we can go to the right tab after certain actions (e.g complete transaction)